src/index.js

```diff
@@ -2,66 +2,38 @@
  * Something to get us started, nothing sacred here, everything can change!
  */
 
-const fs = require('fs');
 const feedQueue = require('./feed-queue');
 const feedWorker = require('./feed-worker');
+const wikiFeed = require('./wiki-feed-parser');
 
 // Start the web server
 require('./backend/web/server');
 
-/**
- * Process a string into a list of Objects, each with a feed URL
- * @param {String} lines
- */
-function processFeedUrls(lines) {
-  return (
-    lines
-      // Divide the file into separate lines
-      .split(/\r?\n/)
-      // Basic filtering to remove any ines that don't look like a feed URL
-      .filter(line => line.startsWith('http'))
-      // Convert this into an Object of the form expected by our queue
-      .map(url => ({ url }))
-  );
-}
-
 /**
  * Adds feed URL jobs to the feed queue for processing
  * @param {Array[Object]} feedJobs - list of feed URL jobs to be processed
  */
-async function enqueueFeedJobs(feedJobs) {
-  feedJobs.forEach(async feedJob => {
-    console.log(`Enqueuing Job - ${feedJob.url}`);
-    await feedQueue.add(feedJob, {
-      attempts: 8,
-      backoff: {
-        type: 'exponential',
-        delay: 60 * 1000,
-      },
-      removeOnComplete: true,
-      removeOnFail: true,
+async function enqueueWikiFeed() {
+  wikiFeed.parseData().then(data => {
```

Comment:
Let's use `await` here, and it will look something like this:

```js
async function enqueueWikiFeed() {
  const data = await wikiFeed.parseData()
  Promise.all(data.map(async feedJob => {
    log.info(`Enqueuing feed job for ${feedJob.url}`);
    await feedQueue.add(feedJob, {
       attempts: 8,
        backoff: {
          type: 'exponential',
          delay: 60 * 1000,
        },
        removeOnComplete: true,
        removeOnFail: true,
    })
  }))
  .catch(err => log.error({ err }, 'Error queuing wiki feeds');
}
```

NOTE: you can't do a `forEach` like this with an `async` function, you have to use `Promise.all` and create `Promises` for each